pom.xml

```diff
@@ -229,6 +229,39 @@
                 <artifactId>jackson-mapper-asl</artifactId>
                 <version>1.6.2</version>
             </dependency>
+            <!-- Provides the PortalPropertySourcesPlaceholderConfigurer
+                 that supports global.properties and Jasypt encryption-->
+            <dependency>
+                <groupId>org.jasig.portal</groupId>
+                <artifactId>uPortal-spring</artifactId>
+                <version>5.0.0-M4</version>
```

Comment:
:arrow_up: uPortal Spring 5.0.0-M5 is now out.

Ref: https://search.maven.org/#artifactdetails%7Corg.jasig.portal%7CuPortal-spring%7C5.0.0-M5%7Cjar

Reply:
@ChristianMurphy is correct, but it won't make a difference.

Reply:
Grr.  I meant to make that a property anyhow.   Will fix shortly.